home/tests/test_content_import_export.py

```diff
@@ -319,6 +319,24 @@ def remove_next_prompt(page: DbDict) -> DbDict:
     return page
 
 
+@per_page
+def remove_buttons(page: DbDict) -> DbDict:
+    if "whatsapp_body" in page["fields"]:
+        for body in page["fields"]["whatsapp_body"]:
+            body["value"].pop("buttons", None)
+    return page
+
+
+@per_page
+def remove_button_ids(page: DbDict) -> DbDict:
+    if "whatsapp_body" in page["fields"]:
```

Comment:
Would it make sense to combine this, `normalise_related_page_ids`, and `normalise_body_ids` into a more generic `normalise_block_ids` filter? (I'm happy to give that a try, probably in a separate PR.)

Reply:
Yeah happy for you to give it a try in a separate PR. I don't think it's high priority though, since we've only got 2 instances of us needing to do this (I prefer to wait until we have 3), and it's in tests not production code, but I think blocks are well enough defined in wagtail that we can make something generic.